src/app/CMakeLists.txt

```diff
@@ -161,10 +161,12 @@ endif (GUI AND WIN32)
 target_link_libraries(${QBT_TARGET_NAME} ${QBT_TARGET_LIBRARIES} QtSingleApplication::QtSingleApplication)
 
 if (APPLE)
-    set(qbt_BUNDLE_NAME "${CMAKE_PROJECT_NAME}")
+    set(qbt_BUNDLE_NAME "${QBT_TARGET_NAME}")
+    execute_process(COMMAND bash -c "sed -e \"s,@EXECUTABLE@,${qbt_BUNDLE_NAME},g\" ${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist > ${qBittorrent_BINARY_DIR}/Info.plist.in")
```

Comment:
If there is no other requirements, one may put in in any binary dir (`${qBittorrent_BINARY_DIR}` is fine). However, when inspecting a build directory, it might be helpful to know that any template file from a give subdir will be placed into the same subdir in the binary dir:
```cmake
configure_file(${qBittorrent_SOURCE_DIR}/dist/mac/Info.plist ${qBittorrent_BINARY_DIR}/dist/mac/Info.plist.in @ONLY)
```
But does it make sense to create a bundle with console application at all?